nutils/mesh.py

```diff
@@ -90,15 +90,15 @@ def line(nodes, periodic=False, bnames=None, *, space: str = 'X'):
   geom = function.rootcoords(space, 1)[0] * scale + offset if uniform else domain.basis('std', degree=1, periodic=[]).dot(nodes)
   return domain, geom
 
-def newrectilinear(nodes, periodic=None, bnames=[['left','right'],['bottom','top'],['front','back']]):
+def newrectilinear(nodes, periodic=None, bnames=[['left','right'],['bottom','top'],['front','back']], spaces=None):
```

Comment:
the commit message should say newrectilinear i/o rectilinear

Reply:
Done.